src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs

```diff
@@ -386,6 +398,7 @@ public static unsafe void Parameter_MarshalerOnPointer_ThrowsMarshalDirectiveExc
 
         public static void Parameter_NullICustomMarshaler_ThrowsTypeLoadException()
         {
+            // typeof(System.Runtime.InteropServices.MarshalDirectiveException): Marshaling directives are invalid.
             Assert.Throws<TypeLoadException>(() => NullCustomMarshalerMethod(""));
```

Comment:
I think we just have throw the right exception from the compiler (I commented on the spot).